src/utils.ts

```diff
@@ -36,6 +40,9 @@ export async function getNoteFileFromTitle(vault: Vault, noteTitle: string): Pro
   return null;
 }
 
+/**
+ * @deprecated Use app.vault.getAbstractFileByPath() instead.
+ */
 export const getNotesFromPath = async (vault: Vault, path: string): Promise<TFile[]> => {
```

Comment:
`getNotesFromPath` gets an array of notes from a folder path, it's not the same as `app.vault.getAbstractFileByPath()`.

Reply:
I see. Removed the deprecated annotation. Maybe it's worth renaming the function to be getNotesFromFolder and rewrite the manual matching logic with getAbstractFileByPath. It works with folder as well. I will leave it to a future PR.